### ISSUE TITLE
[Bug]: CloudWatch resources can no longer be refreshed with default ReadOnlyAccess policy

### ISSUE THREAD

**Issue by stevehipwell** [hashicorp/terraform-provider-aws#28422]
### Terraform Core Version

1.3.6

### AWS Provider Version

4.47.0

### Affected Resource(s)

Observed:

- `aws_cloudwatch_log_group`

Expected:

- `aws_cloudwatch_*`

### Expected Behavior

I'd expect to be able to plan and refresh state using the default AWS ReadOnlyAccess policy (`arn:aws:iam::aws:policy/ReadOnlyAccess`).

### Actual Behavior

The state couldn't be refreshed due to the `logs:ListTagsForResource` action not being present in the default AWS ReadOnlyAccess policy (`arn:aws:iam::aws:policy/ReadOnlyAccess`).

### Relevant Error/Panic Output Snippet

```shell
Error: listing tags for CloudWatch Logs Log Group (/aws/eks/eks-8/cluster): AccessDeniedException: User: arn:aws:sts::0000000000000:assumed-role/Terraform/vault-github-Pull_Request-eks-1671188089-0cv6 is not authorized to perform: logs:ListTagsForResource on resource: arn:aws:logs:eu-west-1:0000000000000:log-group:/aws/eks/eks-8/cluster because no session policy allows the logs:ListTagsForResource action
	status code: 400, request id: 5d53270e-1a1f-46fa-91b6-2f068940009a

  with module.eks.module.control_plane.aws_cloudwatch_log_group.default,
  on .terraform/modules/eks/modules/control-plane/logging.tf line 1, in resource "aws_cloudwatch_log_group" "default":
   1: resource "aws_cloudwatch_log_group" "default" {
```

### Terraform Configuration Files

```terraform
resource "aws_cloudwatch_log_group" "default" {
  name              = "/aws/eks/${var.name}/cluster"
  retention_in_days = var.control_plane_log_retention
  kms_key_id        = null

  tags = var.tags
}
```

### Steps to Reproduce

- Create the resource above using an AWS role with write permissions
- Make a change and run a plan using an AWS role with only the default AWS ReadOnlyAccess policy (`arn:aws:iam::aws:policy/ReadOnlyAccess`)

### References

- https://github.com/hashicorp/terraform-provider-aws/issues/28322
- https://github.com/hashicorp/terraform-provider-aws/pull/28359

### Would you like to implement a fix?

No

**Comment by jungseoklee** [comment]
Could you attach `arn:aws:iam::aws:policy/CloudWatchLogsReadOnlyAccess` to IAM role you use as mitigation if you want to keep AWS Provider Version 4.47.0? Otherwise, I think you need to wait till `logs:ListTagsForResource` is part of `arn:aws:iam::aws:policy/ReadOnlyAccess`.

**Comment by stevehipwell** [comment]
@jungseoklee we use the R/O policy ARN for a large number of roles, this is a major breaking change with a high blast radius.

The point here is that if the changes were made knowing the impact then they should have been SemVer major with documentation changes. If the impact was unknown then this is a defect and should be reverted until it can be delivered to spec.

**Comment by jungseoklee** [comment]
@stevehipwell I agree with your point. I just want to share workaround as user.

**Comment by n3mawashi** [comment]
Hi, Came across this, this morning. This is my work around for now.

inline policy for the role.
`
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "logsListTagsForResource",
            "Effect": "Allow",
            "Action": "logs:ListTagsForResource",
            "Resource": "*"
        }
    ]
}
`

**Comment by ewbankkit** [comment]
@stevehipwell Thanks for raising this issue 👏 and apologies that the change of tagging APIs caused problems.
We have reverted `aws_cloudwatch_log_group` to use the log group-specific APIs via https://github.com/hashicorp/terraform-provider-aws/pull/28492, which will be included in **v4.49.0** of the Terraform AWS Provider, likely to be released in the first week of the New Year (January 5th 2023).
